packages/commons/engine_util.js

```diff
@@ -581,7 +579,7 @@ function extractJSONPath(doc, expr) {
   let results;
 
   try {
-    results = jsonpath(expr, doc);
+    results = jsonpath({ path: expr, json: doc, wrap: false });
```

Comment:
Setting `wrap` to `false` is also a breaking change. Maybe that could be configurable as an option too.

Reply:
I have updated this to be backwards compatible